website/blog/2022-07-27-getting-started-with-the-dbt-semantic-layer.md

```diff
@@ -0,0 +1,165 @@
+---
+title: "Getting Started With The dbt Semantic Layer"
+description: "There's been a lot of discussion around dbt Metrics and what they are. Callum Mccann clarifies what the Semantic Layer is, what is available today, and how one can already begin to use parts of it!"
+slug: getting-started-with-the-dbt-semantic-layer
+
+authors: [callum_mccann]
+
+tags: [dbt product updates]
+hide_table_of_contents: false
+
+date: 2022-07-27
+is_featured: true
+---
+
+# Getting Started With The dbt Semantic Layer
+
+> TLDR: The Semantic Layer is made up of a combination of open-source and SaaS offerings and is going to change how you consume and define metrics.
+
+
+In recent months, we’ve seen a lot of excitement (and some understandable confusion) in the community around what the dbt Semantic Layer is. This is completely understandable, as elements of this “layer” are currently available while others aren’t. To those who aren’t following this saga with the intensity of [someone watching their investments on the crypto market](https://mobile.twitter.com/scannergr1/status/1536198701215109122/photo/1), we hope this blog can provide clarification on the following things:
+
+1. What is the dbt semantic layer?
+2. How do I use it?
+3. What is publicly available now?
+4. What is still in development?
+
+With that, lets get into it!
+
+<!--truncate-->
+
+> Some of you might be confused - you’ve seen us talk about the Metrics Layer in the past and now we’re talking about the Semantic Layer. Well, what was previously called the Metrics Layer has been re-branded to the “Semantic Layer” in order to better reflect our long term plans. Today’s functionality focus largely on metrics but tomorrow’s will expand to be a programmatic interface to the DAG, to better understand entities and what models represent, and much more. Drew’s post about the future of the Semantic Layer explains this far better.
+
+## What is the dbt Semantic Layer?
+
+The dbt semantic layer is a combination of several components - some of these are part of `dbt-core`, some part of `dbt Cloud` (our SaaS offering), and some are net new functionality. They all [combine together like Voltron](https://www.youtube.com/watch?v=5rPSLQxMT8w) to create a single experience through which business users can query data in a context that is familiar to them — metrics. And the best part is that they can do it in systems they are already comfortable using.
```

Comment:
Couple thoughts on this section:
- We start by saying it's a collection of components - wondering if we can lead with _what it allows you to do._ 
- Thinking we can also draw a bit clearer of a line between Semantic Layer (the larger product space) and Metrics (the first use case for the Semantic Layer)
- Something like "the Semantic Layer allows you to query objects in your dbt project from a wide range of BI and analytics facing tools. Metrics is the first use case for the Semantic Layer - we'll walk through exactly what this looks like in practice later on in this post"

Reply:
> ## What is the dbt Semantic Layer?

The dbt Semantic Layer is a new part of dbt to help improve precision and consistency while expanding flexibility and capability in the modern data stack. Our maestro of metrics, Drew Banin, released a blog post detailing the vision of where we're going. The first use case that we are addressing is one that most practicioners **and** stakeholders are familiar with - metrics.

Under the hood, the dbt Semantic layer is collection of several components - some of these are part of dbt Core, some part of dbt Cloud, and some are net new functionality. They all [combine together like Voltron](https://www.youtube.com/watch?v=5rPSLQxMT8w) to create a single experience through which business users can query data in the context of the metric that is most familiar to them. And the best part is that they can do it in systems they are already comfortable using.

Reply:
@jasnonaz thoughts on edits?